### PR TITLE
Fix some incorrect links

### PR DESCRIPTION
**Pull request by slateny**
(some found via `make linkcheck`)

Related to https://github.com/python/devguide/issues/681

### CODE REVIEW

**Review by hugovk — APPROVED** [review]
Thank you!

**Comment by JelleZijlstra** [comment]
There is a merge conflict

**Comment by slateny** [comment]
Merge conflict was due to the whitespace PR being merged which touched the same section. Fixed it, and also changed some things as per the comments in the GH issue

File: buildworker.rst [hunk 1]
```diff
@@ -177,7 +177,7 @@ For Windows:
 
     * Alternatively (note: don't do both!), set up the worker
       service as described in the `buildbot documentation
-      <http://trac.buildbot.net/wiki/RunningBuildbotOnWindows#Service>`_.
+      <https://web.archive.org/web/20201113084744/http://trac.buildbot.net/wiki/RunningBuildbotOnWindows>`_.
```
Comment (ezio-melotti) [inline buildworker.rst:180]: These instructions seem obsolete, this link seems up to date:
```suggestion
      <https://docs.buildbot.net/current/manual/installation/requirements.html#windows-support>`_.
```

**Review by ezio-melotti — CHANGES_REQUESTED** [review]

**Review by ezio-melotti — APPROVED** [review]

**Pull request merged**